src/AppBundle/Service/OrderManager.php

```diff
@@ -142,7 +142,10 @@ public function authorizePayment(OrderInterface $order)
         $stripeOptions = array();
 
         if (!is_null($stripeAccount)) {
+
             if($restaurantPaysStripeFee) {
+                // needed only when using direct charges (the charge is linked to the restaurant's Stripe account)
+                $stripePayment->setStripeUserId($stripeAccount->getStripeUserId());
```

Comment:
This solves https://github.com/coopcycle/coopcycle-web/issues/398 perfectly, well thought!